### PR TITLE
Calculator: Milestone 4 to Develop

### PR DESCRIPTION
**Pull request by praz99**
In milestone 4, I have implemented `calculate` and `operate` functions.

> The `operate` function takes two different numbers and a mathematical symbol (+, -, x, /) as arguments and performs an operation on the passed numbers according to the given symbol.
> The `calculate` function takes an object and a symbol as the arguments and perform some specific operations with the help of the `operate` function.

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    }
+    return next * -1;
+  }
+
+  if (btnName === '%') {
+    if (total) {
+      return operate(total, 100, '/');
+    }
+    return operate(next, 100, '/');
+  }
```
Comment (MuriloRoque) [inline src/logic/calculate.js]: - [ ] I suggest you put this in your `operate.js` module, and use the `mod` function from `Big.js`.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      answer = first - second;
+      break;
+
+    case 'x':
+      answer = first * second;
+      break;
+
+    case '/':
+      answer = first / second;
+      break;
```
Comment (MuriloRoque) [inline src/logic/operate.js]: - [ ] Please use `Big.js` functions `minus`, `mul` and `div` to make those operations, since you are using the library.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes Requested :red_circle: 

### Hello @praz99 :wave: 

### You have done a very good job in this milestone, there are only some minor changes you need to make before moving on. Plase check my comments in your code.

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**